src/git/install.sh

```diff
@@ -235,7 +273,28 @@ echo "Downloading source for ${GIT_VERSION}..."
 curl -sL https://github.com/git/git/archive/v${GIT_VERSION}.tar.gz | tar -xzC /tmp 2>&1
 echo "Building..."
 cd /tmp/git-${GIT_VERSION}
-make -s USE_LIBPCRE=YesPlease prefix=/usr/local sysconfdir=/etc all && make -s USE_LIBPCRE=YesPlease prefix=/usr/local sysconfdir=/etc install 2>&1
+if [ "${ADJUSTED_ID}" = "alpine" ]; then
+    # ref. <https://github.com/alpinelinux/aports/blob/32ac93ffb642031b88ba8639fbb3abb324169dea/main/git/APKBUILD#L126>
+    git_options=("prefix=/usr")
```

Comment:
Should we use `/usr/local` ? We ran into issues last year, see https://github.com/devcontainers/features/pull/395